CHANGES/1047.feature

```diff
@@ -0,0 +1 @@
+Verify if a manifest already exists locally when syncing a remote repo.
```

Comment:
```suggestion
Decreased the number of pull requests sent to a remote registry when syncing content.
```

Reply:
Can we get a mix of both somehow? Let me make a suggestion:
```suggestion
Added a check if a manifest already exists locally to decrease the number of downloads from a remote registry when syncing content.
```